ui/src/submissions/seminars/components/SeminarForm.jsx

```diff
@@ -208,7 +209,16 @@ function SeminarForm({ values }) {
       <ArrayOf
         values={values}
         name="literature_records"
-        label="Related paper(s)"
+        label={
+          <LabelWithHelp
+            label={
+              <Tooltip title="If the seminar refers to an INSPIRE paper, please fill in the link.">
```

Comment:
help `Tooltip` is already part of `LabelWithHelp`, so you just need `label="Related paper(s)"`